src/runtime/wasm.ts

```diff
@@ -148,7 +150,7 @@ export class WasmParticle extends Particle {
       // Memory setup
       memory: this.memory,
       __memory_base: 1024,
-      table: new WebAssembly.Table({initial: 35, maximum: 35, element: 'anyfunc'}),
+      table: new WebAssembly.Table({initial: 43, maximum: 43, element: 'anyfunc'}),
```

Comment:
This is something to address in the (near) future. At the moment it's derived from the wasm binary itself, and by "derived" I mean WebAssembly throws an error whenever I change the number of virtual functions in the C++ code and I update this to the number in the error.

I don't actually know how you're supposed to correctly determine this value on the fly. Worst case, we do some pre-flight analysis on wasm modules before trying to instantiate them.

I haven't commented much yet because this is all very early code.